src/main/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapper.java

```diff
@@ -23,6 +27,8 @@ public String toJson(Object input) {
         try {
             return objectMapper.writeValueAsString(input);
         } catch (JsonProcessingException e) {
+            logger.error(e.getMessage(), e);
```

Comment:
I think this will cause it to get double logged as a RuntimeException is thrown just below this which will get logged as well
REQ